scripts/templates/generate_coverage_report

```diff
@@ -125,24 +129,38 @@ if [[ "$1" == "--codecov" ]]; then
       echo "WARNING: No CODECOV_TOKEN defined. Upload to CodeCov might fail"
    fi
 
-   # Remove other files to be sure CodeCov uses only the coverage.info file
-   rm -f lcov.info
-   remove_gcda_files
-
    # Upload report to CodeCov
+   if [ ! -d build/tmp ]; then
+      mkdir build
```

Comment:
Same as above. Use mkdir -p "$BUILD/tmp"

Reply:
Shouldn't it be "$BUILD"/tmp?

Reply:
Absolutely not. Scripting languages like Bash, Perl, PHP and many others (in particular shells), have been designed to expand variables inside strings. In case the next char matches [A-Za-z0-9_], then the correct syntax would be "${BUILD}something". The weird syntax `"$BUILD"/tmp` might work on Bash, but it's generally unportable.

Reply:
Ok, thanks for the answer!